### ISSUE TITLE
Pinging DNS fails, for no reason?

### ISSUE THREAD

**Issue by MathiasDeWeerdt** [BloodyMods/ServerStarter#22]
As you can see in the screenshot below it claims to not be able to reach google and cloudflare DNS but as you can see I can perfectly reach them.

Not sure if its part of the ServerStarter or something custom inside all the mods 6.

![image](https://user-images.githubusercontent.com/7256303/89733736-e196fc80-da57-11ea-9e8b-4a27aa4ecd9a.png)

**Comment by BloodWorkXGaming** [comment]
Yeah, this is a know issue. It has something todo with the java sockets unable to ping stuff. Has something to do with permissions and is different on Windows and linux.

Tho im not sure how to fix this:
- One way would be to include a different Socket library which would bloat the executable
- Or leave it as is and rely in permissions being set correctly
- ping example.com for example with a get request
- call the Operating System ping command (not reliable but might work best in 99% of the cases)

**Comment by MathiasDeWeerdt** [comment]
> Yeah, this is a know issue. It has something todo with the java sockets unable to ping stuff. Has something to do with permissions and is different on Windows and linux.
> 
> Tho im not sure how to fix this:
> 
> * One way would be to include a different Socket library which would bloat the executable
> * Or leave it as is and rely in permissions being set correctly
> * ping example.com for example with a get request
> * call the Operating System ping command (not reliable but might work best in 99% of the cases)

Yeah, running it as root seems to be working perfectly fine.

**Comment by MathiasDeWeerdt** [comment]
A suggestion:
Move the ping part to the bash / bat file and pass the result as an argument to the jar. If its needed in there, or only there for a warning, you could leave it out entirely.

**Comment by BloodWorkXGaming** [comment]
This should be fixed now as by trying a get request to google.com and example.com
